pkg/printer/enclave.go

```diff
@@ -286,8 +286,15 @@ func Secrets(secrets map[string]models.ComputedSecret, secretsToPrint []string,
 		return
 	}
 
-	headers := []string{"name", "value"}
+	headers := []string{"name"}
+	if vistypes {
+		headers = append(headers, "vistype")
```

Comment:
Here too, torn on whether to call this "vistype" or "visibility"

Reply:
:nod: - same as above here for me too, but brevity is arguably even more important here since `Raw Visibility` is a lot of characters.